src/index.html

```diff
@@ -1,15 +1,168 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>
+      HTML Form
+    </title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <form action = "https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="field">
+        <legend>
+          Personal information
+        </legend>
+        <div class="input">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
          >
```
поправь немного везде где нежно

Reply:
готово